aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group/Program.cs

```diff
@@ -12,8 +12,8 @@
     options.UseSqlite($"Data Source={Path.Join(AppContext.BaseDirectory, "WebMinRouteGroup.db")}");
 });
 
+builder.Services.AddAuthentication().AddJwtBearer();
```

Comment:
Is this supported in .NET 6? If not, then may need to have separate code samples for .NET 6 and .NET 7.

Reply:
This already is in the 7.0-samples directory. Groups are not supported by .NET 6 either. This is only referenced from .NET 7 and greater docs.